File: Python/Maths/Check prime number.py
num = int(input("Enter a positive number: "))
flag = True
if num <= 1:
    flag = False
else:
    i = 2
    for i in range(i, num):
        if(num % i == 0):
            flag = False
            break
            i += i

if flag == True:
    print(num, "is a prime number")
else:
    print(num, "isn't a prime number")

File: Java/Maths/Check prime number.java
import java.util.Scanner;
class primeNumber {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a positive number: ");
        int num = input.nextInt();
        
        boolean flag = true;
        if(num <= 1){
            flag = false;
        }else{
            for (int i = 2; i < num; i++) {
                if(num%i == 0){
                    flag = false;
                    break;
                }
            }
        }

        if(flag == true){
            System.out.println(num + " is a prime number");
        }else{
            System.out.println(num + " isn't a prime number");
        }
    }
}